fem/fespace.hpp

```diff
@@ -320,12 +390,36 @@ class FiniteElementSpace
    bool Conforming() const { return mesh->Conforming(); }
    bool Nonconforming() const { return mesh->Nonconforming(); }
 
+   /// Sets the order of the i'th finite element.
+   /** By default, all elements are assumed to be of fec->GetOrder(). Once
+       SetElementOrder is called, the space becomes a variable order space. */
+   void SetElementOrder(int i, int p);
+
+   /// Returns the order of the i'th finite element.
+   /** @note This is the 'order' of the FiniteElementCollection. The actual
+       polynomial degree may differ, see @a GetElementDegree(). */
+   int GetElementOrder(int i) const;
```

Comment:
Is this only the case for RT spaces? If so, that should be spelled out in this comment. Otherwise I find it a bit confusing.

Reply:
Done